### ISSUE TITLE
Remove empty answer box in "The Commutative and Associative Laws" chapter's first card

### ISSUE THREAD

**Issue by srushtirk** [oppia/oppia-android#4277]
**Describe the bug**
An empty answer box appears on the first card when returned to the first card after visiting other cards in the chapter "The Commutative and Associative Laws"

**To Reproduce**
1. Go to Expressions & Equations 
2. Visit the first and second card
3. Go back to the first card from the second card

**Expected behavior**
The empty box should not be visible and needs to be removed from the UI. Only the forward arrow should be visible. 

**Demonstration**
<img width="366" alt="image" src="https://user-images.githubusercontent.com/27875087/160758091-dc4f209f-cb16-46c6-87f5-e1aa2e32e02f.png">

**Environment**
- Device/emulator being used: Pixel 5
- Android or SDK version (e.g. Android 5 or SDK 21): Android 12
- App version (you can get this through system app settings or via the admin controls menu in-app): 0.7-alpha-d1b2827517

**Additional context**

**Comment by prince02765** [comment]
I would like to work on this issue.

**Comment by BenHenning** [comment]
This issue is currently being worked on @prince02765, apologies for the confusion.

**Comment by BenHenning** [comment]
For searching context, this issue was found in 0.7-alpha (MR4).

**Comment by BenHenning** [comment]
@KolliAnitha could you please check that this is still an issue in the latest beta? I thought this was addressed, but looking back through the code we actually have had something to make sure empty feedback is outright never shown. It's possible there's blank HTML in this particular lesson, but it'd be good to verify the issue actually still exists before we investigate in depth.

Edit: Ah, this can't be checked, I think, since we don't have expressions & equations in the latest beta. I think they will be included in the upcoming beta, though, so this may be something we can check as part of that release.

**Comment by adhiamboperes** [comment]
I think this has to to with computing the playerFeatureSet of the StatePlayerRecyclerViewAssembler. A developer working on this should probably investigate https://github.com/oppia/oppia-android/blob/74f39685add3a6633f6a806aae5b3a2eb3668959/app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt#L1077

**Comment by KolliAnitha** [comment]
yes @BenHenning currently we dont have Expressions and Equations topic

**Comment by adhiamboperes** [comment]
This looks like a duplicate of https://github.com/oppia/oppia-android/issues/4287 which was already fixed in https://github.com/oppia/oppia-android/pull/4259.
